spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -676,36 +676,36 @@ In terms of mapping between the MP-JWT claims and role names used in @RolesAllow
 ### Recommendations for Optional Container Integration
 This section describes the expected behaviors for Java EE container APIs other than JAX-RS.
 
-#### javax.security.enterprise.identitystore.IdentityStore.getCallerGroups(CredentialValidationResult)
+#### jakarta.security.enterprise.identitystore.IdentityStore.getCallerGroups(CredentialValidationResult)
 This method should return the set of names found in the "groups" claim in the JWT if it exists, an empty set otherwise.
 
-#### javax.ejb.SessionContext.getCallerPrincipal()
```

Comment:
@Emily-Jiang If some of `jakarta.* -> javax.*` revert updates in TCK are correct then I'd need to update this doc as well to retain `javax.ejb.SessionContext` for ex. Commenting here so that I don't forget

Reply:
I don't understand @sberyozkin . Do you mean that SessionContext will still be used from the javax namespace? It is also converted to jakarta within Jakarta 9;  https://jakarta.ee/specifications/platform/9/apidocs/jakarta/ejb/sessioncontext

Reply:
@rdebusscher The compilation was failing, I'll try @Emily-Jiang's suggestion a bit later on